testsuite/mockoauth-tests/src/test/resources/simplelogger.properties

```diff
@@ -2,5 +2,6 @@
 org.slf4j.simpleLogger.logFile=target/test.log
 org.slf4j.simpleLogger.showDateTime=true
 org.slf4j.simpleLogger.dateTimeFormat=yyyy-MM-dd HH:mm:ss:SSS
+org.slf4j.simpleLogger.log.org.apache.kafka.common.security=TRACE
```

Comment:
Is this intentional change?

Reply:
It is indeed. The new test makes sure the SASL extensions are in fact sent by the SASL client layer by inspecting the log.